tpm.go

```diff
@@ -217,38 +219,39 @@ func (t *TPMConnection) init() error {
 		if err == nil {
 			return ek, nil
 		}
-		if !tpm2.IsResourceUnavailableError(err, ekHandle) {
+		if !tpm2.IsResourceUnavailableError(err, ekHandle) || !secureMode {
```

Comment:
if we are in !secureMode we return the error immediately but then we just ignore it afaict, it probably would be a more readable to turn the error in that case to nil here already